src/m365/outlook/commands/mail/mail-send.ts

```diff
@@ -120,12 +120,12 @@ class OutlookMailSendCommand extends GraphCommand {
 
   public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
     try {
-    
+
```

Comment:
TBH I think we may even remove this empty line after opening the try but lets leave it. I will do it while merging 👍